Update Wavelog_CI_V.ino

tbegerow opened the pull request:
+implement an enhanced debug system
+ Show CAT-Vals in Setup-Page https://github.com/dg9vh/Wavelog_CI_V/pull/8  from  original repository
+ ptt-query now works and is shown on the setup page and json request
+ secure WebIF (in case of use on a public network e.g. HamNet)
+ Code cleaned up a little bit

tbegerow committed:
Add PTT query for HamClock

Add PTT query to XMLRPC-Server for HamClock direct connection

tbegerow committed:
ptt-query not working yet

trying fix ptt-query

tbegerow committed:
Secure WebIF

Secure WebIF with Username/Password

tbegerow committed:
Update Wavelog_CI_V.ino

+implement an enhanced debug system
+ Show CAT-Vals in Setup-Page #8  from  original repository
+ ptt-query now works and is shown on the setup page and json request
+ secure WebIF (in case of use on a public network e.g. HamNet)
+ Code cleaned up a little bit

dg9vh commented:
There are unresolved conflicts i have to check, so it would take some time for this.

tbegerow committed:
Add 0.96-OLED SSD1306

Sketch: Add SSD1306 0.96" OLED Display that shows the PTT status, frequency, mode, and the transmit power
PCB: Add SSD1306 0.96"-OLED and state led for RX/TX

tbegerow committed:
Add LED for PTT-State, display IP Address

+ RG-LED for PTT-State in addition to the display (monochrome)
+ Reduce the size of the PTT status to display the IP address in the last line.
